index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <div class="nav">
+    <navbar id="navbar">
+      <a href="#">About</a>
+      <a href="#">Projects</a>
+      <a href="#">Contacts</a>
+    </navbar>
+    </div>
+    <div id="welcome-section">
+    <h1 >Hey We are Raphael and Fatuma</h1>
+    <p> We are web developers</p>
+    </div>
+    
+    <div id="projects">
+      <h2>These are some of our projects</h2>
+      
+    <div class="project-tile">
+    <img src="calc.png">
+    <h3> JavaScript Calculator</h3>
+    </div>
+
+    <div class="project-tile">
+      <img src="random-quote-machine.png">
+      <h3> Random Quote Machine </h3>
+      </div>
+
+      <div class="project-tile">
+        <img src="map.jpg">
+        <h3>  Map Data Across the Globe  </h3>
+        </div>
+
+        <div class="project-tile">
+          <img src="tic-tac-toe.png">
+          <h3> Tic Tac Toe Game  </h3>
+          </div>
+
+          <div class="project-tile">
+            <img src="tribute.jpg">
+            <h3>  Tribute Page  </h3>
+            </div>
+
+            <div class="project-tile">
+              <img src="wiki.png">
+              <h3>  Wikipedia Viewer </h3>
+              </div>
+
+              <div class="project-tile">
+                <img src="random-quote-machine.png">
+                <h3> Random Quote Machine </h3>
+                </div>
+                <a href="#"></a>
+                <div class="project-tile">
+                  <img src="random-quote-machine.png">
+                  <h3> Random Quote Machine </h3>
+                  </div>
+                </div>
+                <div id="links">
+                  <h2>Let's work together...</h2>
+                 
+                  <a href="github.com">Github</a>
+      
+                </div>
+    <a  id="profile-link" href="" target="_blank"></a>
+    
+    
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
```

Comment:
It would be better if you get rid of these comments , so that you have a clean and easily readable code.